ting_openformat.field.inc

```diff
@@ -438,6 +439,12 @@ function _ting_openformat_parse_element($elements = NULL, $render_array = array(
         );
       }
     }
+    elseif (!$has_search_code && $key === display) {
```

Comment:
Hvad sker der for den display?

Reply:
skulle det have været 'display' ?

Reply:
Hov, mere held end forstand at det virkede...